src/components/FreestarAdSlot/freestarWrapper.js

```diff
@@ -29,10 +55,61 @@ class FreestarWrapper {
       console.info(`%c${title}`, styles, ...msg)
     }
   }
+
+  /* example mapping
+    [
```

Comment:
i pointed this out in my ticket, but imo the mapping file shouldn't have an array at the top level.
```
{
  mappings: [
     ...
   ]
}
```
It's a small change and doing it this way keeps us open to add fields to the api later without breaking clients.

Reply:
wouldnt changes be tied to the mappings? I dont anticipate adding something the the key value config mapping file thats not tied to an actual key value so seems like we shouldnt need the ability to add something else top level withouth changing the main focus of the file no? Also adding something at the top level would require a code change here either way ...

Reply:
It's best practice to return objects instead of lists on apis because you have the option for extensibility in the future. Do we anticipate it now? No. But who knows 6-12 months from now. Maybe someone decides we need a createdAt/updatedAt timestamp for example.

Accessing added fields would require a code change but it becomes a backwards compatible change using this philosophy.

Reply:
Fair enough ill bite ;)